packages/amplify-e2e-core/src/utils/sdk-calls.ts

```diff
@@ -144,6 +145,20 @@ export const getUserPool = async (userpoolId, region) => {
   return res;
 };
 
+export const getIdentityPoolRoles = async (identityPoolId: string, region: string) => {
+  config.update({ region });
```

Comment:
I'd rather pass this in CognitoIdentity ctor. setting this in config (i.e. globally) may affect other clients and introduce cross test dependnecies.

Reply:
done